pkg/registration/spoke/addon/lease_controller.go

```diff
@@ -160,7 +160,7 @@ func (c *managedClusterAddOnLeaseController) syncSingle(ctx context.Context,
 			Type:    addonv1alpha1.ManagedClusterAddOnConditionAvailable,
 			Status:  metav1.ConditionUnknown,
 			Reason:  "ManagedClusterAddOnLeaseNotFound",
-			Message: fmt.Sprintf("The status of %s add-on is unknown.", addOn.Name),
+			Message: fmt.Sprintf("The status of %s add-on is unknown because get lease from hub failed: %v", addOn.Name, err),
```

Comment:
isn't the err in this case is always not found?

Reply:
Could it be a network eror from agent to access hub?

Reply:
That make sense to me now, close the PR.